### PR TITLE
Docs grammar correction

### PR DESCRIPTION
**Pull request by yurhasko**

### CODE REVIEW

File: docs/samples/config_example_az_container_instance.yaml [hunk 1]
```diff
@@ -35,13 +35,15 @@ deploy:
...
+
+# Multiple tags example: 
+# 
 #tags:
 #  locreg-version: "0.1.0"
 #  test: "test"
 
-# By default, loccreg test all resources as managed-by: locreg
-# If you want to turn off tags which is not recommended, you can set it to false like in example below
+# By default, locreg tags all resources as managed-by: locreg
```
Comment (VitaliiStelmakh) [inline docs/samples/config_example_az_container_instance.yaml:46]: The same mistake is in README.md, could you please fix it?

File: docs/configuration.md [hunk 2]
```diff
@@ -19,13 +19,13 @@ deploy:
...
-Each of the parts corresponds to single item that `locreg` create except of tags. 
-The tags part is used to store the tags that are used to tag resources deployed in cloud vendors.
+Each of the parts corresponds to a single item that `locreg` creates except of tags. 
+The `tags` part is used to store the tags that are used to tag cloud resources
 
 
 ## The default values
 Before going further to configuration, **it's important to understand how default values work.** 
-If you want to use defaults, you must specify a type of service for which you want to use defaults. Like in example below:
+If you want to use defaults, you must specify a type of service for which you want to use defaults. Like in this example below:
```
Comment (Maksym-Perehinets) [inline docs/configuration.md:28]: Shouldn't it be "Like in the example below"?

File: docs/samples/config_example_az_container_instance.yaml [hunk 3]
```diff
@@ -35,13 +35,15 @@ deploy:
...
+# If you want to turn off the tags (which is not recommended), you can set it to `false`` like in example below
```
Comment (Maksym-Perehinets) [inline docs/samples/config_example_az_container_instance.yaml:47]: Extra ` sign after false

**Review by Maksym-Perehinets — APPROVED** [review]

File: docs/configuration.md (hunk 2 above)
Comment (yurhasko) [reply docs/configuration.md:28]: Both are fine

**Review by VitaliiStelmakh — APPROVED** [review]

**Pull request merged**